src/ddmd/mars.d

```diff
@@ -2002,11 +2010,11 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                 {
                     if (isdigit(cast(char)p[7]))
                     {
-                        errno = 0;
-                        const level = strtol(p + 7, &p, 10);
-                        if (*p || errno || level > INT_MAX)
+                        const level = parseDigits(p + 7, int.max);
+                        if (level == uint.max)
                             goto Lerror;
-                        DebugCondition.setGlobalLevel(cast(int)level);
```

Comment:
Who else is using this method?

Reply:
See https://github.com/dlang/dmd/pull/7285